app/views/main.hbs

```diff
@@ -42,11 +46,14 @@
     {{! Do not track parameter }}
     <meta name="twitter:dnt" content="on">
 
-    {{! Web Monetization, if a payment pointer is provider (experimental) }}
+    {{! Web Monetization, if a payment pointer is provider (experimental)}}
     {{#config.monetization}}
-    <meta name="monetization" content="{{.}}">
+      <meta name="monetization" content={{WEB_MONETIZATION_PAYMENT_POINTER}}>
```

Comment:
We need this to be rolled back to `{{.}}` and not `{{WEB_MONETIZATION_PAYMENT_POINTER}}`. It's weird, but it's what's required by the Handlebars syntax.